dhis-2/dhis-support/dhis-support-hibernate/src/main/java/org/hisp/dhis/datasource/DatabasePoolUtils.java

```diff
@@ -176,14 +178,19 @@ public ConfigKeyMapper getMapper() {
     }
   }
 
+  private DatabasePoolUtils() {}
+
   public static DataSource createDbPool(PoolConfig config)
       throws PropertyVetoException, SQLException {
     Objects.requireNonNull(config);
 
     DbPoolType dbType = DbPoolType.valueOf(config.dbPoolType.toUpperCase());
+    log.info("Database pool type value is [{}]", dbType);
 
     if (dbType == DbPoolType.C3P0) {
       return createC3p0DbPool(config);
+    } else if (dbType == DbPoolType.UNPOOLED) {
```

Comment:
would using a switch on the enum value be better here (instead of ifs)? It allows the compiler to help us make sure that all DbTypes are handled (ensuring if any further are added that we handle them as well)